.github/workflows/README.md

```diff
@@ -187,3 +205,254 @@ jobs:
     steps:
       - run: echo Bumped Version from ${{ needs.call-workflow-passing-data.outputs.old-tag }} to ${{ needs.call-workflow-passing-data.outputs.release-tag }}
 ```
+
+## Java Gradle Docker
+
+This workflow will build, test and publish a Java Gradle project including a Jib image. Additionally,
```

Comment:
Please replace `Jib image` with everywhere with `tarball iamge`. This is how Jib uses it: https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#build-an-image-tarball